### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by mueller-ma**
Translated many strings, the other will follow, when I see them in the context used in the app

### CODE REVIEW

**Comment by HugoGresse** [comment]
Maybe you can translate the app name in german?

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    <string name="pref_textsize_key" translatable="false">textSize</string>
+    <string-array name="pref_textsize_values" translatable="false">
+        <item>10</item>
+        <item>12</item>
+        <item>14</item>
+        <item>18</item>
+        <item>24</item>
+    </string-array>
+
+    <string name="anecdote_image_transition_name" translatable="false">imageTransitionName</string>
```
Comment (HugoGresse) [inline app/src/main/res/values-de/strings.xml]: maybe remove from the translated string

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    <string name="anecdote_video_transition_name" translatable="false">videoTransitionName</string>
```
Comment (HugoGresse) [inline app/src/main/res/values-de/strings.xml]: maybe remove from the translated string

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        <item>ButterKnife by Jake Wharton http://jakewharton.github.io/butterknife/</item>
+        <item>Glide by Bumptech http://github.com/bumptech/glide</item>
+        <item>PhotoView by Chris Banes http://github.com/chrisbanes/PhotoView</item>
+        <item>SimpleExoPlayer by Hugo Gresse http://github.com/HugoGresse/SimpleExoPlayer</item>
+    </string-array>
+
+    <!-- Strings related to Settings -->
+    <string name="pref_cat_display">Display</string>
+    <string name="pref_rowstriping_title">Quote row striping</string>
+    <string name="pref_rowstriping_summary">Add a background to separate each quotes</string>
```
Comment (HugoGresse) [inline app/src/main/res/values-de/strings.xml]: enable row stripping change the background color for each item, to be mroe readable. (aka %2)

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    <string name="dialog_website_itemperpage_hint">Anzahl der Objekte auf einer Seite</string>
+    <string name="dialog_website_firstpagezero_hint">Wenn die Nummer der ersten Seite 0 ist</string>
+
+    <string name="dialog_websitechooser_title">Wähle deine Favoriten</string>
+
+
+    <!-- About fragment -->
+    <string name="about_title">Über</string>
+    <string name="about_made">Gemacht mit Liebe von</string>
+    <string name="about_made_person" translatable="false">Hugo Gresse http://hugo.gresse.io/</string>
```
Comment (HugoGresse) [inline app/src/main/res/values-de/strings.xml]: I'll update the app once the PR is merged to add the link to the contributor. thanks for the translation!

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    <string name="using" translatable="false">Third-party libraries</string>
+    <string name="error_server_unknown" translatable="false">Something went wrong with the server :(</string>
```
Comment (HugoGresse) [inline app/src/main/res/values-de/strings.xml]: you can remove the translatable, I'll update the app tomorrow to remove it too on master.

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (mueller-ma) [reply app/src/main/res/values-de/strings.xml]: done

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (mueller-ma) [reply app/src/main/res/values-de/strings.xml]: done

File: app/src/main/res/values-de/strings.xml (hunk 5 above)
Comment (mueller-ma) [reply app/src/main/res/values-de/strings.xml]: remove translatable="false" also from "about_libraries"?

**Comment by mueller-ma** [comment]
> Maybe you can translate the app name in german?

This is a problem I sometime have: Should I translate the App title? I don't know any app/service with a translated name, however "Quote" is a generic name, which could be better when translated.
Your app, your decision

**Comment by HugoGresse** [comment]
I'm thinking about reverting the app name to the french one which is Anecdote, as Quote in english is more a "citation" than an interesting fact

**Comment by mueller-ma** [comment]
@HugoGresse I changed the App name to the german translation of Anecdote
and removed non-translatable tag from one string

**Comment by mueller-ma** [comment]
@HugoGresse When you remove some "translatable="false"" tags, you can merge this pr

**Pull request merged**